### PR TITLE
Adding Routing

### PR DESCRIPTION
**Pull request by yosefsobol**
I've added routing to all the screens we build.
The rooting is in the app file.
So that rooting doesn't cause an error because there are no components yet,
I added a temporary file named components which has a demo of all components

### CODE REVIEW

File: server/client/src/App.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import React, { Component } from "react";
 import "./App.css";
 import axios from "axios";
+import { BrowserRouter, Switch, Route,
+  //  Link
+   } from "react-router-dom";
+import {Login,Registration,Logout,CreateOrJoinSimulation,JoinTheSimulation,JoiningError,SimulationLibrary,SimulationLobby,PassiveParticipator,ActiveParticipator,ManagerStartSimulation,ManagerSimulationInAction} from './components'
```
Comment (AviadGigi) [inline server/client/src/App.js]: Space after '{' and before '}'
Space after ','

File: server/client/src/App.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 import React, { Component } from "react";
 import "./App.css";
 import axios from "axios";
+import { BrowserRouter, Switch, Route,
+  //  Link
```
Comment (AviadGigi) [inline server/client/src/App.js]: remove comments from commuted code

File: server/client/src/App.js [hunk 3]
```diff
@@ -27,7 +31,26 @@ class App extends Component {
         <p>Hello Nathan Krasney</p>
```
Comment (AviadGigi) [inline server/client/src/App.js]: use Education Simulator in the mean time :)

File: server/client/src/App.js [hunk 4]
```diff
@@ -27,7 +31,26 @@ class App extends Component {
...
         <button onClick={this.clickHandler}>Access express server !!!!!</button>
         <p>Got : {this.state.data}</p>
-      </div>
+      
+
+<BrowserRouter>
+<Switch>
+    <Route exact path='/Login' component={Login} />
+    <Route exact path='/Registration' component={Registration} />
+    <Route exact path='/Logout' component={Logout} />
```
Comment (AviadGigi) [inline server/client/src/App.js]: I'm not sure this is a separate route.
@netanelhadad  @garageil  - is the user "card" with the logout button covers all the screen or is it a menu like experience ?

File: server/client/src/components.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
```
Comment (AviadGigi) [inline server/client/src/components.js]: Please add a folder "components" under src and move this file inside that folder

Comment (yosefsobol) [reply server/client/src/components.js]: Thanks for all comments.
I made changes accordingly

File: server/client/src/App.js (hunk 1 above)
Comment (yosefsobol) [reply server/client/src/App.js]: Different depending on the comment

File: server/client/src/App.js (hunk 2 above)
Comment (yosefsobol) [reply server/client/src/App.js]: @AviadGigi 
Different depending on the comment

File: server/client/src/App.js (hunk 3 above)
Comment (yosefsobol) [reply server/client/src/App.js]: @AviadGigi 
Different depending on the comment

File: server/client/src/components.js (hunk 5 above)
Comment (yosefsobol) [reply server/client/src/components.js]: @AviadGigi 
Different depending on the comment

File: server/client/src/App.js (hunk 4 above)
Comment (netanelhadad) [reply server/client/src/App.js]: It's a good question. I think is just a "modal" so don't need a route for it.

File: server/client/src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (yosefsobol) [reply server/client/src/App.js]: Yes,
 everyone is in a file called Components inside Components folder

**Pull request merged**